### PR TITLE
Add css to package

### PR DESCRIPTION
**Pull request by jdungan**

### CODE REVIEW

File: src/chrome/manifest_additions.json [hunk 1]
```diff
@@ -10,8 +10,5 @@
         "16": "img/icon16.png",
         "48": "img/icon48.png"
     },
```
Comment (groovecoder) [inline src/chrome/manifest_additions.json]: This `icons` property needs to go into the master `manifest.json` - not just chrome additions.

codesy-addon-icon

Comment (jdungan) [inline src/chrome/manifest_additions.json]: icons now in manifest.json done by 127bbc4

**Comment by jdungan** [comment]
This is needed to support codesy/codesy#371

**Comment by groovecoder** [comment]
This works; though `gulp dev` is sometimes flaky rebuilding the chrome directory? Filing a separate issue.

**Pull request merged**